scripts/cmake/vcpkg_from_gitlab.cmake

```
@@ -70,132 +70,120 @@ This exports the `VCPKG_HEAD_VERSION` variable during head builds.
 include(vcpkg_execute_in_download_mode)
 
 function(vcpkg_from_gitlab)
-    set(oneValueArgs OUT_SOURCE_PATH GITLAB_URL USER REPO REF SHA512 HEAD_REF FILE_DISAMBIGUATOR)
-    set(multipleValuesArgs PATCHES)
-    # parse parameters such that semicolons in options arguments to COMMAND don't get erased
-    cmake_parse_arguments(PARSE_ARGV 0 _vdud "" "${oneValueArgs}" "${multipleValuesArgs}")
+    cmake_parse_arguments(PARSE_ARGV 0 "arg"
+        ""
+        "OUT_SOURCE_PATH;GITLAB_URL;REPO;REF;SHA512;HEAD_REF;FILE_DISAMBIGUATOR"
```

```suggestion
        "OUT_SOURCE_PATH;GITLAB_URL;USER;REPO;REF;SHA512;HEAD_REF;FILE_DISAMBIGUATOR"
```
Noting that there was previously an accepted `USER` argument. I think it's also fine to just let this fall through to the `UNPARSED_ARGUMENTS` case.

---

yeah, given that it didn't do anything, there's no reason to keep it around.